oauth.go

```diff
@@ -0,0 +1,52 @@
+package stripe
+
+// Type of OAuth scope.
+type ScopeType string
+
+// List of possible values for OAuth scopes.
+const (
+  ScopeTypeReadWrite ScopeType = "read_write"
+  ScopeTypeReadOnly  ScopeType = "read_only"
+)
+
+// Type of token. This will always be "bearer."
+type OAuthTokenType string
+
+// List of possible OAuthTokenType values.
+const (
+  OAuthTokenTypeBearer  OAuthTokenType = "bearer"
+)
+
+// Params for creating OAuth AuthorizeURL's.
+type AuthorizeURLParams struct {
+  State           *string
+  ResponseType    *string
+  Scope           *string
+  RedirectURI     *string
+  StripeLanding   *string
+  AlwaysPrompt    *bool
+  Express         *bool
+}
```

Comment:
You're missing `stripe_user`. I am not a fan of implementing this, but @ob-stripe added this to .NET and so we likely will need to do the same in go for consistency
Params: https://github.com/stripe/stripe-dotnet/blob/master/src/Stripe.net/Services/OAuth/OAuthAuthorizeUrlStripeUserOptions.cs
PR: https://github.com/stripe/stripe-dotnet/pull/1542

Reply:
Will do :)